src/components.d.ts

```diff
@@ -75,7 +75,7 @@ export namespace Components {
         "placeholder": string | undefined;
         "required": boolean;
         "setKeepFocusOnReRender": (keepFocus: boolean) => Promise<void>;
-        "setSelectionRange": (start: number, end: number, direction?: Direction | undefined) => Promise<void>;
+        "setSelectionRange": (start: number, end: number, direction?: any) => Promise<void>;
```

Comment:
This change suggest that the function setSelectionRange direction argument has type `any` in `smoothly-input` but there is not such change in `src/component/input/index.tsx`.

I think `npm run start` will restore the component.d.ts file.